examples/startAndConnect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var connect = require('../');
+var startSimulator = require('node-firefox-start-simulator');
+
+startSimulator({}).then(function(simulator) {
```

Comment:
there is a bug in startSimulator https://github.com/mozilla/node-firefox-start-simulator/issues/8